Header header
float32 av_ch1
float32 av_ch2
float32 av_ch3
float32 av_ch4
int32 ae_ch1
int32 ae_ch2
int32 ae_ch3
int32 ae_ch4


